src/Transports/MassTransit.SqlTransport.PostgreSql/Configuration/PostgresSqlTransportConfigurationExtensions.cs

```diff
@@ -17,10 +18,23 @@ public static IServiceCollection AddPostgresMigrationHostedService(this IService
                 {
                     options.CreateDatabase = create;
                     options.DeleteDatabase = delete;
+                    options.CreateInfrastructure = create;
                 });
             services.AddHostedService<SqlTransportMigrationHostedService>();
 
             return services;
         }
+
+        public static IServiceCollection AddPostgresMigrationHostedService(this IServiceCollection services, Action<SqlTransportMigrationOptions> options)
```

Comment:
I think it would make sense to let the existing method call this method, but I don't know what you would prefer here?

Reply:
Yes, that would make complete sense to dry it up. THanks!

Reply:
Done.